app/main.ts

```diff
@@ -111,4 +112,13 @@ function handleIPCEvents() {
 
     return await readExcel(result.filePaths[0], date);
   });
+
+  ipcMain.handle(
+    'saveParts',
+    async (
```

Comment:
That's really weird formatting. There is no reason to not make this arrow function a one liner